modules/local/japsa_capsim.nf

```diff
@@ -0,0 +1,40 @@
+process JAPSA_CAPSIM {
+
+    tag "$meta.id"
+    label 'process_single'
+
+    container "docker.io/vmurigneux/japsa@sha256:ba74e9c844d115f390be62cde1272cfa5e10492512674ea342a4eeec47840f98"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "CAPSIM does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
```

Comment:
In this case, I was inspired by the qiime2 modules in nf-core/ampliseq (e.g., https://github.com/nf-core/ampliseq/blob/2.6.1/modules/local/qiime2_ancom_asv.nf). I'm not sure how to move it to after the script declaration without getting an error.

Reply:
https://github.com/nf-core/modules/issues/3683 see ehre for a description. I am pinging also @d4straub as it might be relevant. Basically just move it down after the script section and also use `error` instead for more graceful quitting :) Does this help?

Reply:
Thanks for pinging me, indeed, that seems very relevant!

Reply:
Yes, that helps. Thanks